frontend/packages/core/src/stories/stepper.stories.tsx

```diff
@@ -36,20 +39,36 @@ const PrimaryTemplate = ({
     setCurStep(prevActiveStep => prevActiveStep - 1);
   };
 
+  const handleStepClick = step => {
+    setCurStep(step);
+    action("handleStepClick")(step);
+  };
+
   const handleReset = () => {
     setCurStep(0);
+    setCompletedSteps({});
+  };
+
+  const handleCompleteStep = () => {
+    setCompletedSteps({ ...completedSteps, [curStep]: true });
   };
 
   return (
     <>
       <Grid container direction={orientation === "horizontal" ? "column" : "row"}>
         <Grid item xs={1}>
-          <Stepper activeStep={curStep} orientation={orientation}>
+          <Stepper
+            activeStep={curStep}
+            orientation={orientation}
+            handleStepClick={handleStepClick}
+            nonLinear={nonLinear}
+            {...(nonLinear ? { completed: completedSteps } : {})}
```

Comment:
Looks like the storybook setup will need to be adjusted since the logic changed.